docs/SPEC.md

```diff
@@ -325,6 +325,15 @@ Interpolation example:
 
 [IMPL#1775](https://github.com/influxdata/flux/issues/1775) Interpolate arbitrary expressions in string literals
 
+String interpolation supports all basic type arbitrary expressions  without requiring casting to a string. 
+Basic types include integer literals, unsigned integer literals, float literals, boolean literals, time literals and duration literals. 
```

Comment:
another nit (I am picky about the wording in the SPEC :D ) I prefer to use only concepts that have been defined in the SPEC so that it can be a self contained document as much as possible. We haven't defined what `casting` or `basic types` are so my ask is reword this to use the `Stringable` type constraint since that is now defined below. Something like:

String interpolation expressions must satisfy the Stringable constraint.